Dockerfile

```diff
@@ -23,7 +23,8 @@ RUN if [ -n "${GITEA_VERSION}" ]; then git checkout "${GITEA_VERSION}"; fi \
  && make clean-all build
 
 FROM alpine:3.13
-LABEL maintainer="maintainers@gitea.io"
+LABEL maintainer="maintainers@gitea.io" \
+    org.opencontainers.image.source="https://github.com/go-gitea/gitea"
```

Comment:
https://code.gitea.io/gitea ?

Reply:
From that url renovate can't extrat the release notes. Thats why i used the github url, where renovate can extract releases and changelog files